dff/messengers/telegram/abstract.py

```diff
@@ -0,0 +1,364 @@
+"""
+Interface
+------------
+This module implements various interfaces for :py:class:`~dff.messengers.telegram.messenger.TelegramMessenger`
+that can be used to interact with the Telegram API.
+"""
+
+from pathlib import Path
+from tempfile import gettempdir
+from typing import Callable, Optional, Sequence
+from pydantic import FilePath
+
+from telegram import (
+    InlineKeyboardButton,
+    InlineKeyboardMarkup,
+    InputMediaAnimation,
+    InputMediaAudio,
+    InputMediaDocument,
+    InputMediaPhoto,
+    InputMediaVideo,
+    Update,
+    Message as TelegramMessage,
+)
+from telegram.ext import Application, ExtBot, MessageHandler, CallbackQueryHandler, ContextTypes
+from telegram.ext.filters import ALL
+
+from dff.messengers.common import MessengerInterface
+from dff.pipeline.types import PipelineRunnerFunction
+from dff.script.core.message import (
+    Animation,
+    Audio,
+    Button,
+    CallbackQuery,
+    Contact,
+    DataAttachment,
+    Document,
+    Image,
+    Invoice,
+    Keyboard,
+    Location,
+    Message,
+    Poll,
+    PollOption,
+    Video,
+)
+
+
+class _AbstractTelegramInterface(MessengerInterface):  # pragma: no cover
+    request_attachments = {Location, Contact, Invoice, Poll, Audio, Video, Animation, Image, Document}
```

Comment:
I meant to sort them to have attachments that are both in `request_attachments` and `response_attachments` go first so that it would be evident which ones are supported by both and which ones are only supported by e.g. requests.